.changeset/young-coins-sniff.md

```diff
@@ -0,0 +1,6 @@
+---
+"@finos/legend-studio": patch
+"@finos/legend-studio-manual-tests": patch
```

Comment:
Can we create 2 changsets. One empty and one with changelog but only for legend-studio?